### ISSUE TITLE
Use `Get-CimInstance` to detect Hyper-V availability

### ISSUE THREAD

**Issue by letmaik** [kubernetes/minikube#13571]
### What Happened?

https://github.com/kubernetes/minikube/blob/9f96701d8f751d5dfce72f2fb762833f4c396747/pkg/minikube/registry/drvs/hyperv/hyperv.go#L94

[`Get-WmiObject`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-wmiobject?view=powershell-5.1) is deprecated and has been superseded by [`Get-CimInstance`](https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/get-ciminstance?view=powershell-7.2) in 2012 with the release of PowerShell 3.0. Replacing it should be easy and allow to run minikube on systems where the deprecated WMI cmdlets have been blocked by policy.

### Attach the log file

-

### Operating System

Windows

### Driver

Hyper-V
